package.json

```diff
@@ -47,16 +50,34 @@
         "AppImage",
         "deb"
       ]
-    }
+    },
+    "publish": [
+      {
```

Comment:
When publishing a release, builds will be uploaded to s3 and github, but the first provider in the list ("generic") will be used for auto-update checks. The github provider automatically detects the project and branch from your local git repo.